src/client/pfs.go

```diff
@@ -234,9 +235,9 @@ func (c APIClient) inspectCommit(repoName string, branchName string, commitID st
 // If `to` and `from` are the same commit, no commits will be returned.
 // `number` determines how many commits are returned.  If `number` is 0,
 // all commits that match the aforementioned criteria are returned.
-func (c APIClient) ListCommit(repo *pfs.Repo, to, from *pfs.Commit, number int64) ([]*pfs.CommitInfo, error) {
```

Comment:
This changes the signature of client.APIClient.ListCommit, which is a breaking API change and hence requires a new major version.

I would recommend adding a new method like `ListCommitsSince` or somesuch _if_ you think you need to add anything to package client.

Reply:
Makes sense. I've removed the changes to the go client and updated the tests to match